rules/kubelet.libsonnet

```diff
@@ -0,0 +1,25 @@
+{
+  _config+:: {
+    kubeletSelector: 'job="kubelet"',
+  },
+
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'kubelet.rules',
+        rules: [
+          {
+            record: 'node_quantile:kubelet_pleg_relist_duration_seconds:histogram_quantile',
```

Comment:
Open to suggestions on naming and if it's worth calculating for the three quartiles or just the one used (0.99).